src/js/MainController.js

```diff
@@ -50,6 +50,7 @@ goog.require('ga_window_service');
       '</span>';
 
       var map = new ol.Map({
+        moveTolerance: 5,
```

Comment:
It's a number of pixels used to detect a pointermove event or not . it's basically the same thinh we had [here](https://github.com/geoadmin/mf-geoadmin3/pull/3913/files#diff-b636fd252d680aa6c7492587ed17061bL232) and [here](https://github.com/geoadmin/mf-geoadmin3/pull/3913/files#diff-66ac86f99915855fde57c0a20d8f6d6bL342)